src/databricks/labs/ucx/queries/assessment/main/01_3_count_table_by_type_location_format.sql

```diff
@@ -0,0 +1,29 @@
+/* --title 'Table counts by type, location and format' --width 4 --height 4 */
+SELECT object_type AS SOURCE_TABLE_TYPE,
```

Comment:
please run `databricks labs lsql fmt` and provide screenshot of successful dashboard render

Reply:
yay, `make fmt` actually detects it
<img width="638" alt="image" src="https://github.com/user-attachments/assets/b384efd5-1e7c-47e8-9968-ec1b06401939">

Reply:
<img width="1067" alt="image" src="https://github.com/user-attachments/assets/06008753-2ba0-4cac-9c2e-cd4d81ff5108">
<img width="1640" alt="image" src="https://github.com/user-attachments/assets/383332a1-e660-41e1-bd74-895ae05181e9">